pulsar-client-cpp/lib/MessageId.cc

```diff
@@ -30,43 +29,51 @@
 
 namespace pulsar {
 
-MessageId::MessageId() : ledgerId_(-1), entryId_(-1), partition_(-1) {}
+class MessageIdImpl {
```

Comment:
Need to move this to a separate header and cc file, makes creating wrapper around message id easy.

Reply:
Sure